docs/docs/repository-settings.md

```diff
@@ -0,0 +1,50 @@
+# Repository settings
+
+All these settings are applied to this repository.
+
+| Setting | Value | Info |
+|-|-|-|
+| Allow merge commits | `enabled` | Default to pull request title and description |
+| Allow squash merging | `disabled` | |
+| Allow rebase merging | `disabled` | |
+| Always suggest updating pull request branches | `enabled` | |
+| Allow auto-merge | `enabled` | |
+| Automatically delete head branches | `enabled` | |
+| Projects | `disabled` | |
+| Issues | `disabled` | |
+| Allow | `enabled` | |
+
+## Add collaborators
+
+Default GitHub teams will be added as collaborator in the repositories granting access to all project members to contribute to the onboarded repositories.
+
+| Team | Role |
+|-|-|
+| [oauth-proxy-contributors](https://github.com/orgs/philips-forks/teams/oauth-proxy-contributors) | Maintain |
+
+## Create protection rules
+
+These are the protection rules for the `main` and `edi-foundation-integration` branches in order to secury and standarize the way of working.
+
+| Rules | Value | Info |
+|-|-|-|
+| Require a pull request before merging | `Enabled` | N/A |
+| Require approvals | `1` | N/A |
+| Dismiss stale pull request approvals when new commits are pushed | `Enabled` | N/A |
+| Require review from Code Owners | `Enabled` | N/A |
+| Allow specified actors to bypass required pull requests | `Disabled` | N/A |
+| Require status checks to pass before merging | `Enabled` | N/A |
+| Require conversation resolution before merging | `Enabled` | N/A |
+| Require signed commits | `Enabled` | N/A |
+| Do not allow bypassing the above settings | `Enabled` | Enforce rules for admin users. |
+| Restrict who can push to matching branches | `Disabled` | N/A |
+| Allow force pushes | `Disabled` | N/A |
+| Allow deletions | `Disabled` | N/A |
+| Require linear history | `Disabled` | N/A |
+| Require merge queue | `Disabled` | N/A |
+| Require deployments to succeed before merging | `Disabled` | N/A |
+| Lock branch | `Disabled` | N/A |
```

Comment:
Are all the rules configured equally between `master` and `edi-foundation-integration`?

Reply:
The only value that was different for them is the "Lock Branch", which is not enforced yet, but since it will be soon I added it separately for each branch.